lib/processLayerStyles.js

```diff
@@ -16,32 +16,53 @@ const PATTERN_FILL_TYPE = {
 }
 
 const processLayerStyles = async (layerStyles, filepath, outputFolder, modifiers) => {
-    let ret = []
+    let styles = []
+    let properties = []
```

Comment:
This should be clearer - is styles == swiftUI and properties == UIKit ? If so, name them to be clearer